src/header_map.rs

```diff
@@ -0,0 +1,49 @@
+//! COSE HeaderMap
```

Comment:
Could you please split this commit in two: one commit moves the header map to a different folder, the other commit adds the COSE Encrypt functionality.

Reply:
just done thanks!